tools/isobuild/compiler.js

```diff
@@ -431,7 +433,7 @@ var compileUnibuild = function (options) {
     // Find the handler for source files with this extension (unless explicitly
     // tagged as a static asset).
     let classification = null;
-    if (! fileOptions.isAsset) {
+    if ((!fileOptions.isAsset) || fileOptions.isAsset === "both") {
       classification = sourceProcessorSet.classifyFilename(
         filename, inputSourceArch.arch);
 
```

Comment:
Is it ever possible for a `both` file to trigger the error on line 488 (where I cannot make a comment apparently)?  I think so, if you explicitly pass in `isAsset: both` on a thing that doesn't actually have a handler for it.  Does the error message make sense in that case?